packages/cli-dxns/src/dxos.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package dxos;
+
+import "google/protobuf/any.proto";
+
+message Kube {
+  string name = 1;
+  string url = 2;
+  string version = 3;
+}
+
+message Service {
+  string name = 1;
+  string type = 2;
+  string version = 3;
+
+  // Possibility of inheritance for services?
+  google.protobuf.Any service = 4;
```

Comment:
I tend to make these = 10 so there is room, but this is probably a bad pattern -- your choice